scrapy/http/request/rpc.py

```diff
@@ -7,9 +7,13 @@
 import xmlrpc.client as xmlrpclib
 from typing import Any, Optional
 
+from defusedxml import xmlrpc
```

Comment:
I think it's better to import `defusedxml.xmlrpc`, as `xmlrpc` is the name of the stdlib module so this can cause confusion (and maybe name clashes if we need to import the stdlib one in the future).

Reply:
Pushed.